### DISCUSSION TITLE
Login doesn't work

### DISCUSSION THREAD

**Discussion by davs634** [prasathmani/tinyfilemanager#1183] [General]
Hello and thank you for this fantastic script! 

I have an installation problem. The login and password don't work "admin / admin@123". Even if I change and re-do the password with the generator, nothing works. This remains on the login page

**Comment by prasathmani** [comment]
what is you PHP version, make sure you have this requirements https://github.com/prasathmani/tinyfilemanager#requirements

**Reply by davs634** [reply]
Hi ! I have PHP version 8.2

**Reply by psmetana** [reply]
I have same problem. PHP version 8.1

**Reply by psmetana** [reply]
> I have same problem. PHP version 8.1

My problem was that I had passwords in $auth_users in quotation marks. Once I put them in apostrophes, it started working.
